runtime/include/j9cfg.h.ftl

```diff
@@ -61,6 +61,7 @@ extern "C" {
 
 #if JAVA_SPEC_VERSION >= 11
 #define J9VM_OPT_VALHALLA_NESTMATES
+#define J9VM_OPT_VALHALLA_VALUE_TYPES
```

Comment:
Did you meant to commit this? This seems like something for testing since I don't think we want to change the default configuration to Value Types enabled at present...